### ISSUE TITLE
Title as multivalued field

### ISSUE THREAD

**Issue by Numerizen** [scholarslab/SolrSearch#127]
Hi and thanks for your work on this plugin.

I would like to propose result sorting, mainly on the title field. 

For some reason, it's defined as a multivalued field in schema.xml, and don't see why. It prevents passing a parameter like "sort title asc"

As I understand it, the title field in Omeka will never have multiple values, so could you tell me if it's safe to reconfigure my Solr instance with this parameter set to "false" ?

Any other solution would of course also be fine.

**Comment by erochest** [comment]
Actually, there can be more than one title. See the screenshots I've attached :smile:

The question is whether we want to pretend like this can't be the case. It seems like sorting on title is a more typical and compelling use case than having more than one title. We'll talk about this and see what we think.

Thanks for bringing it to our attention.

![screen shot 2016-05-13 at 9 26 58 am](https://cloud.githubusercontent.com/assets/43783/15249365/29939aa6-18ed-11e6-8f6e-70bcb000ad14.png)

![screen shot 2016-05-13 at 9 27 09 am](https://cloud.githubusercontent.com/assets/43783/15249370/3090ad58-18ed-11e6-8f31-09a3320b2267.png)

**Comment by Numerizen** [comment]
You're right : it can. But in my case : it won't :D. Making a title a multivalue field in Omeka seems to be an odd choice to me in the first place anyway, but well.

I changed the value in schema.xml and it appears to be harmless. My sorting works.

Thanks for your quick answer. 

Have a nice day.

**Comment by erochest** [comment]
I'm going to open this back up. I think this is an important use case that we need to consider supporting. No promises yet, but I'd like to keep it in mind. Thanks,
